disprel_split.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+
+# Usage: ./disprel.py path/to/folder
+
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+from matplotlib import cm
+import numpy as np
+import sys
+import os.path
+from os.path import join as pjoin
+from scipy.constants import value as constants
+from glob import glob
+import re
+from tqdm import tqdm
+from aux import *
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Plasma Dispersion Processor')
+parser.add_argument('--i', default=None, type=str, help='Path to E-field data')
```

Comment:
Single-letter arguments are usually short arguments, i.e., `-i` instead of `--i`. It is possible to have both: `parser.add_argument('--input', '-i', ...)` The long form should usually be very descriptive to make it easier for others (like `--periodic`).

Reply:
I like the idea of having multiple options. Fixed and added to the new commit.